### PR TITLE
Pretty print

### PR DESCRIPTION
**Pull request by jmercouris**
Create a function to pretty print binding keys.

### CODE REVIEW

**Comment by jmercouris** [comment]
This additionally reverts the old approach.

**Comment by jmercouris** [comment]
I added `cl-change-case` to satisfy the test runner. I don't know how to calculate the recursive build dependencies, and thus did not add them. I am used to this being the job of a package manager...

Yes, it depends on *print-shortcut* being set to a value, I used to let bind it, I guess I should readd that.

I will add a test as well.

File: tests/tests.lisp [hunk 1]
```diff
@@ -483,21 +483,3 @@
...
-     "control-option-shift-command-a"
-     #+linux
-     "Ctrl-Alt-Shift-Super-a"
-     (nkeymaps/core::key->keyspec
-      (nkeymaps:make-key
-       :value "a"
-       :modifiers '("control"
-                    "meta"
-                    "shift"
-                    "super"))))))
```
Comment (Ambrevar) [inline tests/tests.lisp:503]: @jmercouris Why remove this test?  Mistake?

Comment (jmercouris) [reply tests/tests.lisp:503]: I can't remember from so long ago, probably didn't resolve correctly on non Linux

Comment (Ambrevar) [reply tests/tests.lisp:503]: This needs to be worked on, we can't remove tests without further investigation, it highlights something is wrong.

**Pull request merged**